dotnet/src/Functions/Functions.OpenApi/Model/RestApiOperation.cs

```diff
@@ -257,6 +260,31 @@ private string BuildPath(string pathTemplate, IDictionary<string, object?> argum
         return pathTemplate;
     }
 
+    private object? GetArgumentForParameter(IDictionary<string, object?> arguments, RestApiParameter parameter)
+    {
+        // Try to get the parameter value by the argument name.
+        if (!string.IsNullOrEmpty(parameter.ArgumentName) &&
+            arguments.TryGetValue(parameter.ArgumentName!, out object? argument) &&
+            argument is not null)
```

Comment:
Why skip if the argument is null?

I have have explicitly set the argument to be null should that value not be used?